Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -225,7 +225,7 @@ HTMLLinkElement::LinkProcessingOptions HTMLLinkElement::create_link_options()
     options.referrer_policy = ReferrerPolicy::from_string(get_attribute(AttributeNames::referrerpolicy).value_or(""_string)).value_or(ReferrerPolicy::ReferrerPolicy::EmptyString);
     // FIXME: source set                       el's source set
     // base URL                         document's URL
```

Comment:
It's probably worth changing the comment here as well to: "document's document base URL" to match the spec - document's URL is referring to https://dom.spec.whatwg.org/#concept-document-url not https://html.spec.whatwg.org/multipage/urls-and-fetching.html#document-base-url

Reply:
changed